mantidimaging/gui/windows/image_load_dialog/presenter.py

```diff
@@ -110,8 +110,12 @@ def do_update_sample(self):
         self.view.fields["Sample Log"].use = False
 
         for pos in ["Before", "After"]:
-            self.view.fields[f"Flat {pos} Log"].path = find_log_for_image(self.view.fields[f"Flat {pos}"].path_text())
```

Comment:
NITPICK: Given this is already a pretty long method and we are touching it anyway, what are you're thoughts on breaking this out into a separate method to be called called something like: `flat_path_logs` and break into a separate test in `presenter_test.py`?

Reply:
I'm a bit torn on this. I want to refactor, but I'll do it in future PRs

Reply:
That's fine by me! It's why I wanted to get your opinion on this first. I'll approve this and merge in. We should create and issue to refactor and link to this PR for reference.